packages/ember-cli-code-coverage/package.json

```diff
@@ -91,6 +89,18 @@
     "release-it-lerna-changelog": "^2.3.0",
     "sinon": "^4.2.2"
   },
+  "peerDependencies": {
+    "@embroider/compat": "^0.47.0 || ^1.0.0 || ^2.0.0 || ^3.0.0",
```

Comment:
Can you loosen these to make the last `>=3.0.0`? That way if it happens to work with 4.0 or further, we don't have to release just to bump the peer dep?

Reply:
done ! Thank's for the review